### DISCUSSION TITLE
Error after validation: Method Livewire\Redirector::withInput does not exist

### DISCUSSION THREAD

**Discussion by joostvanhoof** [livewire/livewire#3309] [1. Help]
In my component I have set up a rule:

```php
class Example extends Component
{
    public $tab = 'todo';

    protected $rules = [
        'tab' => ['in:todo,done'],
    ];

    protected $queryString = ['tab'];

    public function render()
    {
        $this->validate();
        
        return view('livewire.example');
    }

}
```

If I now change the querystring `?tab=` to something that's not allowed, like `test`, I get `Method Livewire\Redirector::withInput does not exist`

**Comment by joshhanley** [comment]
@joostvanhoof can you share a screenshot of the error that includes the stack trace?

Thanks!

**Comment by joostvanhoof** [comment]
See below. Please let me know if you need more info!

<img width="1226" alt="Screenshot 2021-07-13 at 09 22 42" src="https://user-images.githubusercontent.com/5268113/125409135-e4167a80-e3bb-11eb-9b9e-464174f5f457.png">
